sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala

```diff
@@ -3731,13 +3718,6 @@ private[sql] object QueryCompilationErrors extends QueryErrorsBase with Compilat
     )
   }
 
-  def indeterminateCollationError(): Throwable = {
```

Comment:
Let's don't remove this, because it is supposed to use in the on-going PR: https://github.com/apache/spark/pull/46004

Reply:
Got

Reply:
@MaxGekk Had recover it，PTAL